### PR TITLE
Appending e2e test cases for issue 1450

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

### What problem does this PR solve?

Appending e2e test cases for https://github.com/chaos-mesh/chaos-mesh/issues/1450

### What is changed and how does it work?

This test cases startup 4 pods which divided into 2 partition:

- partition 0: pods 0, 2
- partition 1: pods 1, 3

Then inject network delay between 2 partitions.
Then make assert about network delay **ONLY exists** in `{0, 1}, {0, 3}, {1, 2}, {2, 3}`.

### Checklist

Tests

- [ ] Unit test
- [x] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by YangKeao — DISMISSED** [review]
LGTM

File: test/e2e/chaos/networkchaos/network_peers_crossover.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+			Delay: &v1alpha1.DelaySpec{
+				Latency:     "200ms",
+				Correlation: "25",
+				Jitter:      "0ms",
+			},
+		}
+		testDelayDuration      = pointer.StringPtr("9m")
+		testDelaySchedulerSpec = &v1alpha1.SchedulerSpec{
+			Cron: "@every 10m",
+		}
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/networkchaos/network_peers_crossover.go:68]: seems that we can remove the scheduler config

File: test/e2e/chaos/networkchaos/network_peers_crossover.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	)
+
+	By("injecting network chaos between partition 0 and 1")
+	networkDelay := makeNetworkDelayChaos(
+		ns, "network-chaos-1",
+		map[string]string{"partition": "0"},
+		map[string]string{"partition": "1"}, // no target specified
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/networkchaos/network_peers_crossover.go:75]:
```suggestion
		map[string]string{"partition": "1"},
```

File: test/e2e/chaos/networkchaos/network_peers_crossover.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Chaos Mesh Authors.
```
Comment (WangXiangUSTC) [inline test/e2e/chaos/networkchaos/network_peers_crossover.go:1]:
```suggestion
// Copyright 2021 Chaos Mesh Authors.
```

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

File: test/e2e/chaos/networkchaos/network_peers_crossover.go (hunk 1 above)
Comment (STRRL) [reply test/e2e/chaos/networkchaos/network_peers_crossover.go]: fixed

File: test/e2e/chaos/networkchaos/network_peers_crossover.go (hunk 2 above)
Comment (STRRL) [reply test/e2e/chaos/networkchaos/network_peers_crossover.go]: fixed

File: test/e2e/chaos/networkchaos/network_peers_crossover.go (hunk 3 above)
Comment (STRRL) [reply test/e2e/chaos/networkchaos/network_peers_crossover.go]: fixed

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**